modules/ROOT/pages/amazon/amazon-sns-connector-reference.adoc

```diff
@@ -1304,46 +1301,121 @@ The Unsubscribe action deletes a subscription. If the subscription requires auth
 * SNS:RETRY_EXHAUSTED
 * SNS:PLATFORM_APPLICATION_DISABLED
 
+
+
 == Types
+[[Tls]]
+=== Tls
+
+[%header,cols="20s,25a,30a,15a,10a"]
+|===
+| Field | Type | Description | Default Value | Required
+| Enabled Protocols a| String | A comma-separated list of protocols enabled for this context. |  |
+| Enabled Cipher Suites a| String | A comma-separated list of cipher suites enabled for this context. |  |
+| Trust Store a| <<TrustStore>> | For servers, `Trust Store` contains certificates of the trusted clients. For clients, `Trust Store` contains certificates of the trusted servers. |  |
```

Comment:
Shouldn't have tick marks on Trust Store. Use "For servers, a trust store contains ..."

Reply:
I think we are referring to the parameter here, which is why I have it in backticks.

Reply:
But why repeat the field name over and over?  Someone reading this knows they're looking at the Trust Store field name. It's much more readable to use the lower case form.

Reply:
Good point.